### PR TITLE
weed shell: fs.rm remove file or directory

### PR DESCRIPTION
**Pull request by combineads**
Add rm.fs command.

### CODE REVIEW

File: weed/shell/command_fs_rm.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		request := &filer_pb.DeleteEntryRequest{
+			Directory:            targetDir,
+			Name:                 targetName,
+			IgnoreRecursiveError: true,
+			IsDeleteData:         true,
+			IsRecursive:          true,
+			IsFromOtherCluster:   false,
+			Signatures:           nil,
+		}
+		_, err = client.DeleteEntry(context.Background(), request)
```
Comment (chrislusf) [inline weed/shell/command_fs_rm.go:57]: this just delete the one file or directory meta data.

It does not delete the file chunks, and it does not recursively delete directory children.

Comment (combineads) [reply weed/shell/command_fs_rm.go:57]: If I use IsDeleteData: true, IsRecursive: true , wouldn't data and subdirectories be deleted?

Comment (chrislusf) [reply weed/shell/command_fs_rm.go:57]: sorry, you are correct and my memory is wrong.

**Comment by chrislusf** [comment]
Merged. Please add some checking to see whether the to-be-removed item exists or not. Here is what I did.

```
> fs.rm -h
remove: /-h
```

**Comment by chrislusf** [comment]
Also, adding some support to differentiate directory?
```
fs.rm -r dir
```

**Comment by combineads** [comment]
OK. I'll add -r and -f options.

**Pull request merged**